pages/services/edge-lb/1.2/installing/index.md

```diff
@@ -77,11 +75,7 @@ If you need to deploy a local Universe containing your own set of packages, you
     sudo make base
     ```
 
-3. Obtain the Edge-LB stub universe JSON files from the support downloads site. Note that there are two files required:
-- `stub-universe-edgelb.json`
-- `stub-universe-edgelb-pool.json`
-
-4. To add the JSON definitions to the universe, use the `add-stub-universe.sh` script.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.  
+3. Use `add-stub-universe.sh` script to add the JSON definitions obtained in ## Obtaining package artifacts section to the universe.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.
```

Comment:
The same here regarding the section.

Reply:
Take out the 'Highly available' from here: 
An **Edge-LB pool** can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer.
Its not Highly Available

Reply:
> Its not Highly Available
I do not follow - Edge-LB is HA, if you deploy multiple instances of a single pool.